packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -225,7 +226,13 @@ function ngGenerateTypescriptSDKFn(options: NgGenerateTypescriptSDKCoreSchematic
     };
 
     const generateOperationFinder = async (): Promise<PathObject[]> => {
-      const specification: any = isJson ? tree.readJson(specDefaultPath) : (await import('js-yaml')).load(tree.readText(specDefaultPath));
+      let specification;
```

Comment:
we do not have a rule to avoid implicit `any`?

Reply:
As discussed earlier we set them as `warn` and they will be fixed later